.readthedocs.yaml

```diff
@@ -0,0 +1,28 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  jobs:
+      pre_build:
+        - inv rtd
```

Comment:
Could you give this task a more verbose name like `readthedocs`? This should help everyone to immediately see the purpose.